widgetssdk/src/main/java/com/glia/widgets/call/domain/HandleCallPermissionsUseCase.kt

```diff
@@ -0,0 +1,27 @@
+package com.glia.widgets.call.domain
+
+import com.glia.androidsdk.Engagement
+import com.glia.widgets.permissions.PermissionsGrantedCallback
+import com.glia.widgets.permissions.domain.PermissionsForEngagementMediaTypeUseCase
+import com.glia.widgets.permissions.domain.PermissionsHandlerUseCase
+
+internal class HandleCallPermissionsUseCase(
+    private val permissionsForEngagementMediaTypeUseCase: PermissionsForEngagementMediaTypeUseCase,
```

Comment:
May it be better to name it:
   a)  `GetPermissionsForEngagementMediaTypeUseCase` 
   b)  `GetPermissionsForEngagementTypeUseCase`
   c)  `GetPermissionsForEngagementUseCase`